src/components/postComponent/components/post-geolocation.svelte

```diff
@@ -0,0 +1,89 @@
+<script lang="ts">
+	import { getToastStore } from '@skeletonlabs/skeleton';
+	import Geolocation from 'svelte-geolocation';
+
+	export let coords: [number, number];
+	let getPosition = false;
+	let isLocationActivated = false;
+	let buttonDisabled = false;
+
+	const toastStore = getToastStore();
+	const buttonLoadingText = 'Geolocation wird geladen';
+	const buttonActivationText = 'Geolocation Aktivieren';
+	const buttonActivatedText = 'Geolocation aktiviert';
```

Comment:
Konstanten schreibt man in CAPS als beispielsweise BUTTON_LOADING_TEXT

Reply:
Darüber hinaus mag ich das Wort "Geolocation" nicht. Wollen wir nicht einfach "Standort" nehmen?